css-fonts-and-colors/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Nunito, sans-serif;
+  font-optical-sizing: auto;
+  font-weight: <weight>;
+  font-style: normal;
+}
+
+margin: 0;
+
+section {
+  padding: 1.5rem 0;
+  background-color: lightblue;
+}
+
+h2 {
+  margin-bottom: 3.5rem;
+  font-size: 2rem;
+  text-align: center;
+  font-family: Ubuntu, sans-serif;
+  font-weight: 500;
```

Comment:
Headings should be at least 600 or 700 so that it's bold

Reply:
Thank you I changed it.